examples/faith/pegasus_workflow_conf.ini

```diff
@@ -0,0 +1,137 @@
+[lalapps_inspinj]
+min-mass1 = 50
+max-mass1 = 150
+min-spin1 = 0
+max-spin1 = 1
+min-mass2 = 50
+max-mass2 = 150
+min-spin2 = 0
+max-spin2 = 1
+min-mtotal = 100
+max-mtotal = 300
+enable-spin =
+aligned =
+f-lower = 20
+waveform = IMRPhenomXAS
+disable-milkyway =
+i-distr = uniform
+l-distr = random
+min-distance = 1000
+d-distr = uniform
+max-distance = 1000
+gps-start-time = 1000000000
+gps-end-time = 1000001000
+time-interval = 0.
+time-step = 1.
+seed = 123434
+m-distr = componentMass
+
+[executables]
+lalapps_inspinj = ${which:lalapps_inspinj}
+pycbc_faithsim = ${which:pycbc_faithsim}
+pycbc_faithsim_collect_results = ${which:pycbc_faithsim_collect_results}
+pycbc_faithsim_plots = ${which:pycbc_faithsim_plots}
+pycbc_splitbank = ${which:pycbc_splitbank}
+
+[workflow]
+file-retention-level = all_triggers
+
+[splitbank]
+num_banks = 10
+
+[pycbc_faithsim]
+psd-model = aLIGOZeroDetHighPower
+waveform1-approximant = IMRPhenomXAS
+waveform1-start-frequency = 14
+waveform2-approximant = SEOBNRv4_opt
+waveform2-start-frequency = 14
+filter-low-frequency = 20
+filter-sample-rate = 16384
+filter-waveform-length = 1024
+
+[pycbc_faithsim_plots]
+[pycbc_faithsim_plots-mass1_mass2_time_offset]
+parameter-x = mass1
+parameter-y = mass2
+parameter-z = time_offset
+colormap = viridis
+
+[pycbc_faithsim_plots-mass_ratio_spin1z_match]
+parameter-x = mass_ratio
+parameter-y = spin1z
+parameter-z = match
+colormap = viridis
+vmin = 0
+vmax = 1
+
+[pycbc_faithsim_plots-mass_ratio_spin1_magnitude_match]
+parameter-x = mass_ratio
+parameter-y = spin1_magnitude
+parameter-z = match
+colormap = viridis
+vmin = 0
+vmax = 1
+
+[pycbc_faithsim_plots-mass_ratio_spin2z_match]
+parameter-x = mass_ratio
+parameter-y = spin2z
+parameter-z = match
+colormap = viridis
+vmin = 0
+vmax = 1
+
+[pycbc_faithsim_plots-mass_ratio_spin2_magnitude_match]
+parameter-x = mass_ratio
+parameter-y = spin2_magnitude
+parameter-z = match
+colormap = viridis
+vmin = 0
+vmax = 1
+
+[pycbc_faithsim_plots-mass_ratio_mtotal_match]
+parameter-x = mass_ratio
+parameter-y = total_mass
+parameter-z = match
+colormap = viridis
+vmin = 0
+vmax = 1
+
+[pycbc_faithsim_plots-mass1_mass2_match]
+parameter-x = mass1
+parameter-y = mass2
+parameter-z = match
+colormap = viridis
+vmin = 0
+vmax = 1
+
+[pycbc_faithsim_plots-coa_phase_match_mchirp]
+parameter-x = coa_phase
+parameter-y = match
+parameter-z = mchirp
+colormap = viridis
+ymin=0
+
+[pycbc_faithsim_plots-inclination_match_mchirp]
+parameter-x = inclination
+parameter-y = match
+parameter-z = mchirp
+colormap = viridis
+
+[pycbc_faithsim_plots-inclination_sigma1_mchirp]
+parameter-x = inclination
+parameter-y = sigma1
+parameter-z = mchirp
+colormap = viridis
+
+[pycbc_faithsim_plots-inclination_sigma2_mchirp]
+parameter-x = inclination
+parameter-y = sigma2
+parameter-z = mchirp
+colormap = viridis
+
+[pegasus_profile]
+condor|accounting_group = ligo.dev.o4.cbc.uber.pycbcoffline
+condor|request_disk = 100000
+condor|request_memory = 2048
```

Comment:
Please add 
```
pycbc|primary_site = condorpool_symlink
```
as an option in this section. It will default to this anyway, but this would/could be a good example of where OSG would be useful and being able to just flip that to osg, and have it run there without issue, would be great!